README.md

```diff
@@ -3,7 +3,7 @@ KRF
 
 [![Build Status](https://travis-ci.com/trailofbits/krf.svg?branch=master)](https://travis-ci.com/trailofbits/krf)
 
-KRF is a **K**ernelspace **R**andomized **F**aulter for the Linux kernel.
+KRF is a **K**ernelspace **R**andomized **F**aulter for the Linux and FreeBSD kernels.
```

Comment:
Let's go ahead and remove explicit references to Linux and FreeBSD here and below, since we plan on adding more kernels as well.

We could do this:

```
KRF is a **K**ernelspace **R**andomized **F**aulter.

It currently supports the Linux and FreeBSD kernels.
```